src/main/java/smokefree/aws/rds/secretmanager/SmokefreeConstants.java

```diff
@@ -32,4 +32,9 @@
 	public static final Long MAXIMUM_PLAYGROUNDS_DISTANCE = 100L;
 	public static final Long MAXIMUM_PLAYGROUNDS_ALLOWED = 1000L;
 
+	public static class PlaygroundWorkspace {
```

Comment:
Why is this in a separate class?

Reply:
Its created just to segregate the constants.

Reply:
hmm. seems a bit too over-engineered for me. Your call.

Reply:
Other remarks as I just noticed: Why are these in the secretmanager package?

Reply:
I agree. Even I feel the same and I thought It would be good If I do it during refactoring/renaming packages.

Reply:
ok then leave it for now